sdk-python/agentcube/utils/http.py

```diff
@@ -0,0 +1,44 @@
+"""HTTP session utilities for AgentCube SDK."""
+
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+
+
+def create_session(
+    pool_connections: int = 10,
+    pool_maxsize: int = 10,
+    retry_total: int = 3,
+    retry_backoff_factor: float = 0.5,
+    retry_status_forcelist: tuple = (502, 503, 504),
```

Comment:
should we disable retry for now.  Especially without looking at  what `502, 503, 504` means.